RefactoringsExperiments/src/main/java/org/jetbrains/research/extractMethodExperiments/legacy/DatasetPostprocessor.java

```diff
@@ -83,14 +85,14 @@ public static void fixCsv(int N) throws FileNotFoundException {
             }
         }
 
-        try(FileWriter fileWriter = new FileWriter("ress.csv")) {
+        try (FileWriter fileWriter = new FileWriter("ress.csv")) {
```

Comment:
ress?

Reply:
maybe we should delete unused legacy code?